MekHQ/src/mekhq/campaign/universe/RandomFactionGenerator.java

```diff
@@ -273,6 +287,18 @@ public String getEnemy(Faction employer, boolean useRebels) {
         return "PIR";
     }
 
+    /**
+     * Appends MERC faction to the given enemy map, with approximately a 10% probability
+     */
+    protected void appendMercsToEnemyMap(WeightedMap<Faction> enemyMap) {
+        int mercWeight = 0;
+        for (int key : enemyMap.keySet()) {
+            mercWeight += key;
+        }
+        
+        enemyMap.add((int) (mercWeight / 10), Factions.getInstance().getFaction("MERC"));
```

Comment:
Let's add a `Math.max(1, (int)(mercWeight / 10))`. We've had a few odd bugs with enemies and employers at the fringes of our data.